README.md

```diff
@@ -0,0 +1,19 @@
+![DocumentViewer icon](DocumentViewer_icon.png)DocumentViewer
+=
+
+DocumentViewer is a viewer supporting PDF and DJVU files. 
+It provides many features, making it easy to open and view documents.<br>
+
+<img src="DocumentViewerScreenshot.png" height="360" width="480">
+ 
+## Features include:
+  * Opening a file dialog to load a document
+  * Print document
+  * View panel
+  * Zooming in or out
+  * Fitting the page length or width
+  * Shows thumbnail
+  * Find text
+  * Home panel giving access to other feature
+  * Options for help or to the website
```

Comment:
A link to the project website isn't that much of a feature. How about just:
* Offline user documentation

Also, remove application/DocumentViewer_icon.png from the PR. We have the icon already in the root folder.